### PR TITLE
Update: Project Aurras - MVP - Phase 2

### PR DESCRIPTION
**Pull request by MuhammedIrfan**
Update #814

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks a lot for the amendment. Could you also update the original timeline/estimated duration?

**Comment by MuhammedIrfan** [comment]
> Thanks a lot for the amendment. Could you also update the original timeline/estimated duration?

Sure, I have updated. Thank you.

**Review by Noc2 — DISMISSED** [review]
Thanks for the update.

File: applications/project_aurras_mvp_phase_2.md [hunk 1]
```diff
@@ -237,3 +234,16 @@ We came across the Web3 Grants Program while perusing about the developments in
...
+### 28/12/2022
+
+We have encountered a critical issue where we faced workflows fails due to certain Openwhisk actions timing out hence we took an alternative approach where we have made Polkadot API wasm compatible and included as a plugin to the wasm, due to this the timeline has now been exceeded by several months and has made certain components we developed obsolete.
+
+| Number | Deliverable | Specification | Link |
+| ------------- | ------------- | ------------- | ------------- |
+| 0a. | staking and payout features for [scs/substrate-api-client](https://github.com/scs/substrate-api-client) | staking functionalities and payout example. fixed bugs in staking modules | https://github.com/scs/substrate-api-client/pull/294 |
+
+- **Estimated Duration:** 45 Working days
+- **FTE:**  1
```
Comment (alxs) [inline applications/project_aurras_mvp_phase_2.md:249]: What is this and how is it related to the rest of the project? If you want to deliver it with the next milestone, could you add it to its deliverable table?

**Review by alxs — CHANGES_REQUESTED** [review]
Thanks, but could you also update the estimated duration (in time instead of person days) or just add an estimated delivery date to each milestone? It's not clear when you want to deliver the first milestone since it still says 30 working days.

**Comment by MuhammedIrfan** [comment]
> 



> Thanks, but could you also update the estimated duration (in time, not just person days) or just add an estimated delivery date to each milestone? It's not clear when you want to deliver the first milestone since it still says 30 working days.

Sure @alxs , I will update the application with the estimated delivery date

File: applications/project_aurras_mvp_phase_2.md (hunk 1 above)
Comment (MuhammedIrfan) [reply applications/project_aurras_mvp_phase_2.md:249]: We have used [scs/substrate-api-client](https://github.com/scs/substrate-api-client) as a wasm-compatible lib for the workflow to interact with substrate-based chains. We have made some contributions that would be needed for our use case. We will add this to the deliverables. Thank you

**Comment by Noc2** [comment]
@MuhammedIrfan feel free to ping me once it's updated.

**Comment by MuhammedIrfan** [comment]
> @MuhammedIrfan feel free to ping me once it's updated.

I have updated. Kindly review.

**Review by Noc2 — APPROVED** [review]
Thanks for the update. I'm happy to go ahead with it and share it again with the rest of the team.

**Review by keeganquigley — APPROVED** [review]

**Review by takahser — APPROVED** [review]

**Pull request merged**